blueprints/flexberry-edit-form/files/__root__/routes/__name__/new.js

```diff
@@ -2,5 +2,6 @@ import EditFormNewRoute from 'ember-flexberry/routes/edit-form-new';
 
 export default EditFormNewRoute.extend({
   modelProjection: '<%=modelProjection%>',
-  modelName: '<%=modelName%>'
+  modelName: '<%=modelName%>',
+  templateName: '<%=entityName%>',
```

Comment:
So name of route and template that use for edit form are equal to 'entityName', aren't they?